webapp/src/utils/styles.js

```diff
@@ -0,0 +1,121 @@
+
+            '&:hover': {
+                borderColor: changeOpacity(theme.centerChannelColor, 0.25),
+            },
+        }),
+        option: (provided, state) => ({
+            ...provided,
+            background: state.isFocused ? changeOpacity(theme.centerChannelColor, 0.12) : theme.centerChannelBg,
+            cursor: state.isDisabled ? 'not-allowed' : 'pointer',
+            color: theme.centerChannelColor,
+            '&:hover': state.isDisabled ? {} : {
+                background: changeOpacity(theme.centerChannelColor, 0.12),
+            },
+        }),
+        clearIndicator: (provided) => ({
+            ...provided,
+            width: '34px',
+            color: changeOpacity(theme.centerChannelColor, 0.4),
+            transform: 'scaleX(1.15)',
+            marginRight: '-10px',
+            '&:hover': {
+                color: theme.centerChannelColor,
+            },
+        }),
+        multiValue: (provided) => ({
+            ...provided,
+            background: changeOpacity(theme.centerChannelColor, 0.15),
+        }),
+        multiValueLabel: (provided) => ({
+            ...provided,
+            color: theme.centerChannelColor,
+            paddingBottom: '4px',
+            paddingLeft: '8px',
+            fontSize: '90%',
+        }),
+        multiValueRemove: (provided) => ({
+            ...provided,
+            transform: 'translateX(-2px) scaleX(1.15)',
+            color: changeOpacity(theme.centerChannelColor, 0.4),
+            '&:hover': {
+                background: 'transparent',
+            },
+        }),
+        menu: (provided) => ({
+            ...provided,
+            color: theme.centerChannelColor,
+            background: theme.centerChannelBg,
+            border: '1px solid ' + changeOpacity(theme.centerChannelColor, 0.2),
+            borderRadius: '0 0 2px 2px',
+            boxShadow: changeOpacity(theme.centerChannelColor, 0.2) + ' 1px 3px 12px',
+            marginTop: '4px',
+        }),
+        input: (provided) => ({
+            ...provided,
+            color: theme.centerChannelColor,
+        }),
+        placeholder: (provided) => ({
+            ...provided,
+            color: theme.centerChannelColor,
+        }),
+        dropdownIndicator: (provided) => ({
+            ...provided,
+
+            '&:hover': {
+                color: theme.centerChannelColor,
+            },
+        }),
+        singleValue: (provided) => ({
+            ...provided,
+            color: theme.centerChannelColor,
+        }),
+        indicatorSeparator: (provided) => ({
+            ...provided,
+            display: 'none',
+        }),
+    };
+};
+
+export const getLabelFontColor = (hexcolor) => {
+    const r = parseInt(hexcolor.substr(0, 2), 16);
+    const g = parseInt(hexcolor.substr(2, 2), 16);
+    const b = parseInt(hexcolor.substr(4, 2), 16);
+    const yiq = ((r * 299) + (g * 587) + (b * 114)) / 1000;
+    return (yiq >= 128) ? 'black' : 'white';
```

Comment:
if you are trying to pass color from here then use hex code `#ffffff` for white and `#000000` for black

remember always try using only `hex` or `rgb, rgba` color codes on the website for better compatibility